README.md

```diff
@@ -31,65 +20,105 @@ cm <- ConnectionManager({
 	"retry": false,
 	"stayConnected": false
 });
-uart <- hardware.uart12;
-logs <- UartLogger(uart);
 
-wq <- WifiQueue(cm, wifiList, logs);
+wq <- WifiQueue(cm, WIFIQUEUE_HIDDEN | WIFIQUEUE_OPEN)
+
+wq
+	.onConnect(function() {
+		logger.log("WifiQueue connected to " + imp.getssid());
+	})
+	.onFail(function() {
+		logger.error("WifiQueue failed to connect");
+	});
```

Comment:
This semi-colon shouldn't be here.
Also, you haven't defined logger but have used it.